app/controllers/merchant/dashboard_controller.rb

```diff
@@ -2,7 +2,27 @@ class Merchant::DashboardController < Merchant::BaseController
   before_action :require_merchant
 
   def index
+    @pending_orders = []
+    @pending_items = []
+    @pending_item_orders = []
     @merchant = Merchant.find(current_user.merchant_id)
+    @orders = Order.all
```

Comment:
Items have merchant_id
Items connect to ItemOrders
ItemOrders connect to Orders

Possible Pseudocode for AR for this joins?
Items.group(merchant_id == current_user.merchant_id).joins(ItemOrders).selects(orders)